tests/testthat/test_featsel_filters.R

```diff
@@ -72,3 +72,16 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
     more.args = list("randomForestSRC_var.select" = c(method = "vh.imp"))))
   )
 })
+
+
```

Comment:
this empty line is not needed

Reply:
Wow! Thank you so much. I have added my name to the list of contributors.